engine/engine/content/builtins/assets/meshes/quad.dae

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<COLLADA xmlns="http://www.collada.org/2005/11/COLLADASchema" version="1.4.1">
+  <asset><contributor><author></author><authoring_tool>FBX COLLADA exporter</authoring_tool><comments></comments></contributor><created>2012-10-10T12:34:45Z</created><keywords></keywords><modified>2012-10-10T12:34:45Z</modified><revision></revision><subject></subject><title></title><unit meter="1.000000" name="centimeter"></unit><up_axis>Y_UP</up_axis></asset>
```

Comment:
```
<unit meter="1.000000" name="centimeter"
```

Is there a downside to having this model in centimeters?

Reply:
No idea! The file is from the template-basic-3d repo.

Reply:
Changed this to meters